### PR TITLE
DOC-1118: RS-Add command-line options to install doc.

### PR DESCRIPTION
**Pull request by lanceleonard**
- Initial draft

### CODE REVIEW

**Comment by lanceleonard** [comment]
A [staged preview](https://docs.redis.com/staging/jira-doc-1118/rs/installing-upgrading/#install-command-line-options) is available.

**Review by kaitlynmichael — APPROVED** [review]

File: content/rs/installing-upgrading/_index.md [hunk 1]
```diff
@@ -180,11 +180,34 @@ To install without answering the installation questions, either:
...
+
+## Install command line-options
+
+Run `./install.sh --help` to view command-line options supported by the install script.
+
+At this time, the following options are supported:
+
+| Option | Description |
+|--------|-------------|
+| `-y` | Automatically answers `yes` to all install prompts prompts, accepting all default values |
```
Comment (rrelledge) [inline content/rs/installing-upgrading/_index.md:201]:
```suggestion
| `-y` | Automatically answers `yes` to all install prompts, accepting all default values |
```

File: content/rs/installing-upgrading/_index.md [hunk 2]
```diff
@@ -180,11 +180,34 @@ To install without answering the installation questions, either:
...
+| <nobr>`-c <answer file>`</nobr> | Specify answer file used to respond to install prompts |
+| <nobr>`-s <socket dir>`</s> | Specify directory for redislabs unix sockets  _(new installs only)_|
+| <nobr>`--install-dir <dir>`</nobr> | Specifies installation directory _(new installs only)_| 
+| <nobr>`--config-dir <dir>` | Configuration file directory _(new installs only)_|
+| <nobr>`--var-dir <dir>`</nobr> | Var dir used for installation _(new installs only)_|
+| <nobr>`--os-user <user>`| Operating system user account associated with install (default: `redislabs`; _new installs only)_|
+|<nobr>`--os-group <group>` | operating system group associated with install (default: `redislabs`; _new installs only)_ |
```
Comment (rrelledge) [inline content/rs/installing-upgrading/_index.md:208]:
```suggestion
|<nobr>`--os-group <group>` | Operating system group associated with install (default: `redislabs`; _new installs only)_ |
```

**Review by rrelledge — APPROVED** [review]
Looks good to me. I just left two small suggestions.

File: content/rs/installing-upgrading/_index.md (hunk 1 above)
Comment (lanceleonard) [reply content/rs/installing-upgrading/_index.md:201]: Fixed.

File: content/rs/installing-upgrading/_index.md (hunk 2 above)
Comment (lanceleonard) [reply content/rs/installing-upgrading/_index.md:208]: Fixed.

**Pull request merged**